spring-boot-test-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -45,6 +45,15 @@ org.springframework.boot.autoconfigure.jdbc.JdbcTemplateAutoConfiguration,\
 org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration,\
 org.springframework.boot.autoconfigure.transaction.TransactionAutoConfiguration
 
+# AutoConfigureJooqc auto-configuration imports
```

Comment:
There is maybe a case for something that's configuring a `DataSource` with a `PlatformTransactionManager` and the necessary database migration stuff. That duplication is quite annoying.